lib/practice_exercises.rb

```diff
@@ -1,13 +1,42 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1) - I don't feel confident about this though because of the creation of the comparison array, which I think would double the memory needed.
 def is_palindrome(string)
```

Comment:
Since you're using `.split` you're creating a new array of all the letters... so the space complexity is O(n).  Otherwise 👍